src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -567,65 +567,117 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
 def write_shard_metadata(
     observer: str, obs_data: pd.DataFrame, obs_dir: Path, seed_ext: str
```

Comment:
Basically what is happening in this function is:
-we get counts for each noise type for each state - year combination
-we also aggregate the shard data to get counts at the national level and an aggregation for all years